backend/src/main/java/com/woowacourse/levellog/application/LevellogService.java

```diff
@@ -25,6 +27,11 @@ public class LevellogService {
     private final MemberRepository memberRepository;
 
     public Long save(final Long authorId, final Long groupId, final LevellogRequest request) {
+        final Optional<Levellog> possibleLevellog = levellogRepository.findByAuthorIdAndTeamId(authorId, groupId);
+        if (possibleLevellog.isPresent()) {
+            throw new LevellogAlreadyExistException();
+        }
```

Comment:
```suggestion
        levellogRepository.findByAuthorIdAndTeamId(authorId, groupId)
                .ifPresent(it -> {
                    throw new LevellogAlreadyExistException();
                });
```
이런식으로 Optional 메서드를 활용해도 좋을거 같네요!🙂